packages/garbo/src/combat.ts

```diff
@@ -108,6 +110,25 @@ export class Macro extends StrictMacro {
     return new Macro().trySingAlong();
   }
 
+  ifHeavyRainsWanderer(macro: Macro): Macro {
+    return this.externalIf(
+      myPath() === HeavyRains.path,
+      Macro.if_([...HeavyRains.wanderers], macro),
+    );
+  }
+
+  static ifHeavyRainsWanderer(macro: Macro): Macro {
+    return new Macro().ifHeavyRainsWanderer(macro);
+  }
+
+  ifWanderer(macro: Macro): Macro {
```

Comment:
I think this name is too general given that we use `wanderer` to refer to things like sausage goblins and deliberately encountered embezzlers in garbo

Reply:
Hmm. ifPassiveWanderer?